app/src/main/res/resource/values-de/strings.xml

```diff
@@ -26,17 +26,17 @@
     <string name="search">SUCHE</string>
     <string name="follow">FOLGEN</string>
     <string name="following">FOLGT</string>
-    <string name="my_follow">Meine Folgungen</string>
+    <string name="my_follow">Anhänger</string>
     <string name="all">ALLES</string>
     <string name="stats">Status</string>
     <string name="by">Von</string>
     <string name="on">An</string>
     <string name="now">Jetzt</string>
 
-    <string name="liked">Gemocht</string>
+    <string name="liked">Favoriten</string>
     <string name="collected">Gesammelt</string>
-    <string name="followed">Folgende</string>
-    <string name="released">Entlassen</string>
+    <string name="followed">Gefolgt</string>
+    <string name="released">Veröffentlicht</string>
     <string name="published">Publiziert</string>
     <string name="curated">Curated</string>
```

Comment:
What about "Kuratiert" (as strange as it sounds…). It's the literal translation: https://dict.leo.org/englisch-deutsch/Curated https://de.wiktionary.org/wiki/kuratieren

A more common one would maybe be "Gesammelt"… Depends on how this is used.
**Edit**: Oh, just see it is already used for "collected", so we need to know the difference between these two strings.